content/chainguard/chainguard-images/features/ca-docs/custom-assembly-chainctl.md

```diff
@@ -78,33 +78,95 @@ EOF
 
 ```
 Applying build config to custom-assembly
-  (*v1.CustomOverlay)(Inverse(protocmp.Transform, protocmp.Message{
-  	"@type": s"chainguard.platform.registry.CustomOverlay",
-  	"contents": protocmp.Message{
-  		"@type": s"chainguard.platform.registry.ImageContents",
-  		"packages": []string{
-- 			"wolfi-base",
-+ 			"bash",
-- 			"go",
-+ 			"curl",
-+ 			"mysql",
-  		},
-  	},
-  }))
+  (*v1.CustomOverlay)(Inverse(protocmp.Transform, protocmp.Message{
+      "@type": s"chainguard.platform.registry.CustomOverlay",
+      "contents": protocmp.Message{
+          "@type": s"chainguard.platform.registry.ImageContents",
+          "packages": []string{
+-             "wolfi-base",
++             "bash",
+-             "go",
++             "curl",
++             "mysql",
+          },
+      },
+  }))
 
 Are you sure?
 Do you want to continue? [y,N]: 
 ```
 
 This approach is useful in cases where you would prefer to avoid any kind of interactivity, as in a CI/CD or other automation system.
 
+### Using the `--save-as` option
+
+> **Note**: The Custom Assembly `--save-as` option is currently in beta. [Contact us](https://www.chainguard.dev/contact?utm_source=cg-academy&utm_medium=referral&utm_campaign=dev-enablement) if you'd like to access this feature during the beta period.
+
+When customizing a Chainguard Container with Custom Assembly, you have the option to either customize the image itself or create a new image based on the original with your customizations applied to it.
+
+For example, say your organization has access to Chainguard's [`node` container image](https://images.chainguard.dev/directory/image/node/versions). If you use Custom Assembly to customize the `node` image without creating a new image, then the customizations applied to it will also apply to any users in your organization that are already consuming the image. Anyone who runs `docker pull cgr.dev/example.come/node` will download the customized image instead of the original, uncustomized one.
+
+By creating a new image with Custom Assembly, you can customize the image without impacting any of the users or workflows already consuming it. You could also create multiple customized images based on the `node` container image to support specific functions.
+
+To use `chainctl` to create new customized container images with Custom Assembly, you must include the `--save-as` option, like this:
+
+```shell
+chainctl image repo build edit --parent $ORGANIZATION --repo $CONTAINER --save-as $NEW_NAME
+```
+
+The following example command will create a new image named `custom-node` after applying the customizations:
+
+```shell
+chainctl image repo build edit --parent example.com --repo node --save-as custom-node
+```
+
+Following this example, the new container image would be accessible from the following URL:
```

Comment:
I think I would chance the tense / mood here, above you use "the following example .... will"

Maybe I would do "Once you run this example ... the new container is ... " or if you want to be tentative you could say "If you run this example ... it would be ..."